recipes/pyschism/meta.yaml

```diff
@@ -12,23 +11,21 @@ source:
   patches:
     - setup.patch
 
-
 build:
   noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python >=3.9
```

Comment:
This should stay the same. That variable is defined for you. Please follow only the suggested changes.